doc/api/util.md

```diff
@@ -167,6 +167,14 @@ corresponding argument. Supported placeholders are:
 * `%f` - Floating point value.
 * `%j` - JSON.  Replaced with the string `'[Circular]'` if the argument
 contains circular references.
+* `%o` - Object. A string representation of an object
+  with generic JavaScript object formatting.
+  Similar to `util.inspect()` with options `{ showHidden: true, depth: 4, showProxy: true }`.
```

Comment:
What's the default for `depth`? I think we shouldn't change its default.

I'm okay with `showProxy`, but some people may interpret it as violating a security contract (where Proxies are supposed to be transparent)…